tcms/testruns/models.py

```diff
@@ -268,14 +266,24 @@ class TestExecutionStatus(TCMSActionModel):
         'FAIL': '#cc0000'
     }
 
-    complete_status_names = (PASSED, 'ERROR', FAILED, WAIVED)
-    failure_status_names = ('ERROR', FAILED, BLOCKED)
-    passed_status_names = (PASSED, WAIVED)
-    idle_status_names = (IDLE,)
+    from django.utils.translation import gettext_lazy as _
+    POSITIVE = 'P', _('Positive')
+    NEGATIVE = 'N', _('Negative')
+    NEUTRAL = 'NN', _('Neutral')
+
     chart_status_names = ('pass', 'fail', 'idle', 'other')
 
     id = models.AutoField(db_column='case_run_status_id', primary_key=True)
     name = models.CharField(max_length=60, blank=True, unique=True)
+    status = models.CharField(
+        max_length=60,
+        choices=[POSITIVE, NEGATIVE, NEUTRAL]
+    )
+    is_completed = models.BooleanField()
+    chart_name = models.CharField(max_length=60)
```

Comment:
This is not needed. Charts will be modified to display all statuses with possible enable/disable which ones to be shown and or grouppings based on weight.